tests/rptest/test_suite_quick.yml

```diff
@@ -19,5 +19,9 @@ quick:
   - tests/compacted_topic_verifier_test.py
   - tests/compaction_recovery_test.py
   - tests/wasm_identity_test.py
+  - tests/pandaproxy_test.py::PandaProxyTest.test_consumer_group_binary_v2
+  - tests/pandaproxy_test.py::PandaProxyTest.test_consumer_group_json_v2
+  - tests/pandaproxy_test.py::PandaProxyTest.test_subscribe_consumer_validation
+  - tests/pandaproxy_test.py::PandaProxyTest.test_remove_consumer_validation
```

Comment:
why are the "simplest wasm test" (wording from commit message) excluded

Reply:
removing this commit